themes/dco/js/expertiseCountQuery.js

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
do we need expertiseCountQuery.js? Seems same as subjectAreas.js, just a single variable defined.